src/solvers/adaptive_time_solver.C

```diff
@@ -90,24 +102,22 @@ void AdaptiveTimeSolver::advance_timestep ()
   // The first access of advance_timestep happens via solve, not user code
   // It is used here to store any initial conditions data
   if (!first_solve)
-    {
-      // We call advance_timestep in user code after solve, so any solutions
-      // we will be storing will be for the next time instance
-      _system.time += _system.deltat;
-    }
-    else
-    {
-      // We are here because of a call to advance_timestep that happens
-      // via solve, the very first solve. All we are doing here is storing
-      // the initial condition. The actual solution computed via this solve
-      // will be stored when we call advance_timestep in the user's timestep loop
-      first_solve = false;
-    }
-
-  // If the user has attached a memory or file solution history object
-  // to the solver, this will store the current solution indexed with
-  // the current time
-  solution_history->store(false, _system.time);
+  {
```

Comment:
Fix the indentation here?  It wasn't quite according to the libMesh style the first time either (that else did indeed need to be moved back 2 spaces) but we don't need further changes to make it further off still.